lib/domgen/sync/templates/pg_reseed_procs.sql.erb

```diff
@@ -0,0 +1,10 @@
+/* DO NOT EDIT: File is auto-generated */
+<%= banner("Reseeding '#{data_module.name}' tables procedures") %>
+<% data_module.entities.select{|e|e.sync? && !e.abstract?}.each do |entity| -%>
+CREATE OR REPLACE FUNCTION <%= data_module.sql.quoted_schema %>.<%= data_module.sql.dialect.quote("spReseed#{entity.name}") %>() RETURNS VOID AS '
+    BEGIN
```

Comment:
Current sql server code compacts the table prior to restarting the sequence. Also does this restart this with max value in table?

Also you can probably use entity.primary_key.sql.sequence.qualified_sequence_name to get sequence name